plotly/tests/test_core/test_tools/test_TraceFactory.py

```diff
@@ -0,0 +1,89 @@
+from unittest import TestCase
+from plotly.graph_objs import graph_objs, Line
+from plotly.exceptions import PlotlyError
+
+import plotly.tools as tls
+import math
+from nose.tools import raises
+
+
+class TestQuiver(TestCase):
+
+    def test_unequal_xy_length(self):
+
+        # check: PlotlyError if x and y are not the same length
+
+        kwargs = {'x': [1, 2], 'y': [1], 'u': [1, 2], 'v': [1, 2]}
+        self.assertRaises(PlotlyError, tls.TraceFactory.create_quiver,
```

Comment:
:cow2: since we're now dropping python 2.6 support, you can use this as a context manager!

```
with self.assertRaises(PlotlyError):
    tls.TraceFactory.create_quiver(**kwargs)
```

just for future reference, the former syntax can get pretty ugly